translate/storage/test_mozilla_lang.py

```diff
@@ -9,6 +9,21 @@
 from translate.storage import mozilla_lang, test_base
 
 
+@pytest.mark.parametrize(
+    "orig, stripped", [
+        ("", ""),
+        ("String", "String"),
+        ("String {ok}", "String"),
+        ("String {OK}", "String"),
+        ("String{ok}", "String"),
+        ("String{OK}", "String"),
+        ("String  {ok}", "String"),
```

Comment:
`"String  {OK}"` is missing. Also I would add checks for strings with whitespace after `{ok}`